ui/src/components/Row/Row.vue

```diff
@@ -138,7 +139,7 @@ export default {
 
       this.defaultOptions.style.maxWidth = this.defaultOptions.boxedWidth + 'px'
       this.defaultOptions.style.width = this.defaultOptions.boxedWidth + 'px'
-      this.boxedInFullWidthStatus = this.deviceWidth <= this.defaultOptions.boxedWidth
+      this.boxedInFullWidthStatus = this.deviceWidth <= this.defaultOptions.boxedWidth + Number(this.rowOptions.paddingOfBoxedInFullWidth.split('px')[0])
```

Comment:
get paddingOfBoxedInFullWidth number from a method and handle split and zero index from there